pages/pricing/pricing.css

```diff
@@ -0,0 +1,567 @@
+:root{
+    --dark-grey: #f5f5f5;
+}
+
+html{
+    font-size: 62.5%;
+}
+
+body{
+    font-family: "DM Sans";
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100vw;
+    min-height: 100vh;
+    background-color: var(--pure-white);
+    overflow-x: hidden;
+}
+
+.first-container, .pricing{
+    justify-content: center;
+    align-items: start;
+     width: 37.5rem;
+     background-color: var(--pure-black);
+     color: var(--pure-white);
+}
+.image-hero{
+    background: url(assets/mobile/hero.jpg);
+    background-repeat: no-repeat;
+    width: 37.5rem;
+    height: 30rem;
+}
+.linear{
+    position: absolute;
+    width: 12.8rem;
+    height: 0.6rem;
+    background: var(--main-accent);
+    margin-left: 3rem;
+    margin-top: -0.6rem;
+}
+.pricing{
+    padding: 7.5rem 2.8rem 7.2rem 2.9rem;
+    gap: 1.6rem;
+}
+.pricing h1{
+    font-size: 3.2rem;
+    line-height: 4rem;
+    letter-spacing: 0.3rem;
+}
+.pricing p{
+    font-weight: 400;
+    font-size: 1.5rem;
+    line-height: 2.5rem;
+    mix-blend-mode: normal;
+    opacity: 0.6;
+}
+.second-container, .slicer-div, .same, .plan{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.second-container, .first-container, .pricing, .same{
+    flex-direction: column;
+}
+.first-container, .pricing, .check-icons, .last-container, .arrow{
+    display: flex;
+}
+h1{
+    font-weight: 700;
+}
+.slicer-div{
+    margin-top: 6.4rem;
+    flex-direction: row;
+    gap: 3.2rem;
+    width: 25.5rem;
+}
+.slicer-div p{
+    font-weight: 700;
+    font-size: 1.8rem;
+    line-height: 2.5rem;
```

Comment:
It's better if you write line-height in ems because they should be relative to font-size property in size